README.md

```diff
@@ -98,7 +98,7 @@ We hope VDP can also enrich the open-source communities in a way to bring more p
 
 - **macOS or Linux** - VDP works on macOS or Linux, but does not support Windows.
 
-- **Docker and Docker Compose** - VDP uses Docker Compose (compose file version: `3.9`) to run all services at local. Please install [Docker](https://docs.docker.com/get-docker/) and [Docker-compose](https://docs.docker.com/compose/install/) before using VDP.
+- **Docker and Docker Compose** - VDP uses Docker Compose (compose file version: `3.9`) to run all services at local. Please install [Docker](https://docs.docker.com/get-docker/) and [Docker-compose](https://docs.docker.com/compose/install/) before using VDP. Be aware to use Docker-compose v1 with command `docker-compose` rather than v2 `docker compose`.
```

Comment:
Either `docker-compose` or `docker compose` works out of the box for [Docker Desktop version 4.4.2+](https://www.docker.com/blog/announcing-compose-v2-general-availability). Could you please double check if you have the [compatibility mode ](https://docs.docker.com/desktop/linux/#general) enabled? 

https://docs.docker.com/desktop/linux/#general

Reply:
It works, thanks!